action/protocol/poll/protocol.go

```diff
@@ -122,7 +122,9 @@ func NewProtocol(
 	if cfg.Consensus.Scheme != config.RollDPoSScheme {
 		return nil, nil
 	}
-	if !genesisConfig.EnableGravityChainVoting || electionCommittee == nil || genesisConfig.GravityChainStartHeight == 0 {
+
+	if !genesisConfig.EnableGravityChainVoting || genesisConfig.GravityChainStartHeight == 0 ||
```

Comment:
remove `genesisConfig.GravityChainStartHeight == 0`

Reply:
so why remove it? it's in current code